### ISSUE TITLE
[CI]: dont deploy when indexing failed

### ISSUE THREAD

**Issue by revolunet** [SocialGouv/code-du-travail-numerique#1230]
S'assurer que les index ES existent bien avant de déployer :)

**Comment by revolunet** [comment]
Also, we should ensure `wait-for-elasticsearch` is finished before deploying update

**Comment by revolunet** [comment]
We should add something like this before deploying 

```sh
#!/bin/sh

# wait 120s for ES index to be ready
function index_should_exist() {
  INDEX_NAME=$1
  URL="${ELASTICSEARCH_URL}/${INDEX_NAME}/_search?q=hello"
  retry=120; # 5s * (12 * 10) = 10min
  echo "Check index '${INDEX_NAME}' on '${ELASTICSEARCH_URL}'"
  while
    ! curl -sS --fail "${URL}" && [[ $(( retry-- )) -gt 0 ]];
  do echo "Waiting for index '${INDEX_NAME}' to be available($(( retry )))" ; sleep 5s ; done ;

  [[ $(( retry )) -lt 1 ]] && exit 128;
}

index_should_exist "code_du_travail_numerique"

index_should_exist "conventions_collectives"

exit 0
```

**Comment by lionelB** [comment]
should be fixed by #1753 since we use alias index
